newIDE/app/src/ObjectsList/index.js

```diff
@@ -780,9 +780,13 @@ const ObjectsList = React.forwardRef<Props, ObjectsListInterface>(
                     height={height}
                     getItemName={getObjectWithContextName}
                     getItemThumbnail={getObjectThumbnail}
-                    getItemId={(objectWithContext, index) => {
-                      return 'object-item-' + index;
-                    }}
+                    getItemId={(objectWithContext, index) =>
+                      'object-item-' + index
+                    }
+                    getItemData={(objectWithContext, index) => ({
+                      objectName: objectWithContext.object.getName(),
+                      global: String(objectWithContext.global),
```

Comment:
maybe use `toString()` instead? I'm not sure using a String constructor like this is recommended